### DISCUSSION TITLE
Coretan pertama gue di forum ini.

### DISCUSSION THREAD

**Discussion by deaafrizal** [deaafrizal/forum#2] [The Lounge]
Cuma pengen curhat juga sih...

Kenapa ya masih banyak di kita yang tanya-tanya hal kek macem ginian ke programmer? please take a look closer bro 👇

![image](https://github.com/deaaprizal/forum/assets/43339356/438da9c3-620f-47bc-b459-318e1ac08bf5)

Yang gue resahin pertama, video youtube yang gue bikin itu apaan dan pertanyaan orang apaan kek gak ada konteks banget gituloh. Ya gak sih?

sorry nih kagak gue sensor orangnya, orang dia komen secara publik 🤣.

Agak laennya adalah ini kan jaman dah rada kemana gituloh, walaupun indonesia negara berkembang tapi kan otak kita mah tetep harus maju. Masa iya otak cuma ngembang tapi kagak maju2, meledak kelessss...

**Comment by deeninsom** [comment]
Maklum bang, mungkin aja lagi pingin kayak si MARK, trus jadi kaya awokawok

**Reply by deaafrizal** [reply]
sebelum gue jawab dia mungkin gue dah punya pulau duluan keknya wkwkk

**Reply by deeninsom** [reply]
masih banyak lo bang orang" yang ngga sebegitu paham tentang IT pertanyaanya aneh", apalagi mahasiswa Sistem Informasi kayak aku, gilak masih disuruh hack ig, trus disuruh hack wa, apa ngga pusing jadinya wkwk

**Comment by pututdev** [comment]
assalamualaikum bang dea, izin join forum bang wkwk

**Reply by deaafrizal** [reply]
yoi bre silahkan nongki chill di forum ini ya 👌

**Reply by IrnandaNanda** [reply]
Gimana caranya join ya

**Comment by pikupik** [comment]
punten mang izin join

**Reply by deaafrizal** [reply]
yoyow santei aja bray 👌 selamat nongki

**Comment by Gayoyo** [comment]
![Screenshot (359)](https://github.com/deaaprizal/forum/assets/171170565/19ded69f-2bee-4ded-b832-c9a17e7e84eb)
bang ini extensi apa bang
